pwndbg/heap/ptmalloc.py

```diff
@@ -49,6 +49,8 @@ def __init__(self, addr):
         self._bk = None
         self._fd_nextsize = None
         self._bk_nextsize = None
+        self._arena = arena
+        self._is_top_chunk = None
 
         # TODO key, REVEAL_PTR
```

Comment:
If this TODO is still relevant, can we extend it to include more about what is it for/about or what we would like to achieve with it? :P